### PR TITLE
some more tests

### PR DESCRIPTION
**Pull request by grota**

### CODE REVIEW

File: tests/Unit/GitlabActionsCommandTest.php [hunk 1]
```diff
@@ -41,6 +52,34 @@ public function testExecute() {
       ->assertOutputContains('"project_id": 12345');
   }
 
+  public function testMissingVars() {
```
Comment (Bladedu) [inline tests/Unit/GitlabActionsCommandTest.php:60]: Maybe a small DocBlock would be great here (for CS).

File: tests/Unit/GitlabActionsCommandTest.php [hunk 2]
```diff
@@ -41,6 +52,34 @@ public function testExecute() {
...
+      putenv("$variable=");
+      TestCommand::for(new GitlabActionsCommand($client->reveal()))
+        ->execute('mergeRequests.all')
+        ->assertOutputContains("\"$variable\" env variable is not set.")
+        ->assertStatusCode(1);
+      putenv("$variable=something");
+    }
+  }
+
+  public function test() {
```
Comment (Bladedu) [inline tests/Unit/GitlabActionsCommandTest.php]: Can we specialize a bit more the test method name?

**Review by Bladedu — CHANGES_REQUESTED** [review]

File: tests/Unit/GitlabActionsCommandTest.php (hunk 2 above)
Comment (grota) [reply tests/Unit/GitlabActionsCommandTest.php]: fixed

File: tests/Unit/GitlabActionsCommandTest.php (hunk 1 above)
Comment (grota) [reply tests/Unit/GitlabActionsCommandTest.php:60]: fixed

**Review by Bladedu — APPROVED** [review]
LGTM.

**Pull request merged**